src/modules/dashboard/components/ColonyMembers/ColonyMembers.tsx

```diff
@@ -108,6 +149,20 @@ const ColonyMembers = () => {
     });
   }, [openPermissionManagementDialog, colonyData, isVotingExtensionEnabled]);
 
+  const selectedDomain = colonyData?.processedColony?.domains?.find(
+    ({ ethDomainId }) => ethDomainId === selectedDomainId,
+  );
+
+  const nativeToken = colonyData?.processedColony?.tokens.find(
+    ({ address }) =>
+      address === colonyData?.processedColony?.nativeTokenAddress,
+  );
+
+  const formattedTotalDomainRep = getFormattedTokenValue(
+    new Decimal(totalReputation?.userReputation || '0').abs().toString(),
+    nativeToken?.decimals || DEFAULT_TOKEN_DECIMALS,
```

Comment:
The number of decimals in the reputation, does it really depend on which token we use? That doesn't seem logical to me... It's not a token value, shouldn't it use the same decimals across all colonies? What am I missing?

Reply:
As a disclaimer, I'm following the example of places where Mr.Dan used the `getFormattedTokenValue` to format reputation.

With that being said, this makes sense to me because, from what I understand, when a user is paid a certain amount of tokens, an equal amount of token is generated as reputation points, right?

So If we pay someone `25`, which without formatting it could be `25x10^18`,  `25x10^15`. or  `25x10^23` just to give some examples, we can expect that the generated rep points will be the same as the paid amount.

I could be wrong tho, as this is just my understanding from seeing how the reputation oracle generates the exact amount of tokens that the user is paid.